### PR TITLE
Added page routing for projects and external links, refactored media …

### PR DESCRIPTION
**Pull request by Nkethey422**
…queries, and included netflix clone video background.

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -101,7 +111,7 @@ button {
   }
 
   .work,
-  .about {
+  .aboutbut {
```
Comment (cmcgill2) [inline src/App.css]: Not sure about this rename

File: src/components/Navbar.js [hunk 2]
```diff
@@ -2,17 +2,33 @@ import React from "react";
...
   return (
     <div className="container__navbar">
       <div className="navbar__buttons">
         <div className="wo">
-          <button>WORK</button>
+          <button className="work">
+            <NavLink
+              to="/"
+              exact
+              style={{ textDecoration: "none", color: "inherit" }}
```
Comment (cmcgill2) [inline src/components/Navbar.js]: I would move away from inline styles

File: src/components/Navbar.js [hunk 3]
```diff
@@ -2,17 +2,33 @@ import React from "react";
...
+            </NavLink>
+          </button>
         </div>
         <div className="ab">
-          <button>ABOUT</button>
+          <button className="aboutbut">
+            <NavLink
+              to="/about"
+              exact
+              style={{ textDecoration: "none", color: "inherit" }}
```
Comment (cmcgill2) [inline src/components/Navbar.js]: ditto inline styles

File: src/components/Netflix.js [hunk 4]
```diff
@@ -1,16 +1,36 @@
...
+          position: "absolute",
+          width: "100%",
+          height: "100%",
+          objectFit: "cover",
+          opactiy: "0.2",
+          zIndex: "-100",
+        }}
+      >
+        <source src={Net} type="video/mp4" />
+      </video>
```
Comment (cmcgill2) [inline src/components/Netflix.js]: Could probably factor this component out into its own component and pass the info in as props. seems like it is pretty similar everywhere

File: src/components/Titles.js [hunk 5]
```diff
@@ -1,22 +1,40 @@
...
-//             <span>IMAGE MAKER</span>
-//           </h2>
-//       </div>
-//     </div>
-//   );
-// }
+function Titles({ clickHandler, active }) {
+  return (
+    <div className="container__titles">
+      <>
```
Comment (cmcgill2) [inline src/components/Titles.js]: I don't think you need this

File: src/components/Titles.js [hunk 6]
```diff
@@ -1,22 +1,40 @@
...
+        <h2>
+          <NavLink
+            to="/Netflix"
+            exact
+            style={{ textDecoration: "none", color: "inherit" }}
+          >
+            <span>NETFLIX CLONE</span>
+          </NavLink>
+        </h2>
```
Comment (cmcgill2) [inline src/components/Titles.js]: Should make this a component and pass in the props

File: src/index.js [hunk 7]
```diff
@@ -2,16 +2,17 @@ import React from "react";
 import ReactDOM from "react-dom";
 import App from "./App";
 import "./index.css";
-import reportWebVitals from "./reportWebVitals";
+// import reportWebVitals from "./reportWebVitals";
```
Comment (cmcgill2) [inline src/index.js]: this can be deleted

File: src/index.js [hunk 8]
```diff
@@ -2,16 +2,17 @@ import React from "react";
...
-  </React.StrictMode>,
+  </Router>,
   document.getElementById("root")
 );
 
 // If you want to start measuring performance in your app, pass a function
 // to log results (for example: reportWebVitals(console.log))
 // or send to an analytics endpoint. Learn more: https://bit.ly/CRA-vitals
-reportWebVitals();
+// reportWebVitals();
```
Comment (cmcgill2) [inline src/index.js]: this can be deleted

**Review by cmcgill2 — CHANGES_REQUESTED** [review]
Nice! could use a little cleanup around the components and inline styles

File: src/App.css (hunk 1 above)
Comment (Nkethey422) [reply src/App.css]: Changed .aboutbut back to .about

File: src/components/Navbar.js (hunk 2 above)
Comment (Nkethey422) [reply src/components/Navbar.js]: Added classname "nav__link" to styles and removed all inline styles.

File: src/components/Navbar.js (hunk 3 above)
Comment (Nkethey422) [reply src/components/Navbar.js]: ditto

File: src/components/Netflix.js (hunk 4 above)
Comment (Nkethey422) [reply src/components/Netflix.js]: video element inline styling removed, added to App.css. Hesitant to make it its own component.

File: src/components/Titles.js (hunk 5 above)
Comment (Nkethey422) [reply src/components/Titles.js]: removed.

File: src/components/Titles.js (hunk 6 above)
Comment (Nkethey422) [reply src/components/Titles.js]: Created array of objects (h2__titles) and rendered h2/NavLink elements utilizing .map() function.

Comment (Nkethey422) [reply src/components/Titles.js]: Used the same method in App.js for routing, also surrounded each component with divs of className page in order to utilize onEnter and onExit events from react-transition-group.

File: src/index.js (hunk 7 above)
Comment (Nkethey422) [reply src/index.js]: removed

File: src/index.js (hunk 8 above)
Comment (Nkethey422) [reply src/index.js]: removed

**Pull request merged**